mapie/classification.py

```diff
@@ -183,7 +183,7 @@ def __init__(
         method: str = "score",
         cv: Optional[Union[int, str, BaseCrossValidator]] = None,
         n_jobs: Optional[int] = None,
-        random_state: Optional[Union[int, np.random.RandomState]] = None,
+        random_state: Optional[Union[int, np.random.RandomState]] = 1,
```

Comment:
Warning, scikit-learn convention is to let this default value to `None`

Reply:
OK, I initially set the random_state to an integer to have reproductible random numbers in `_add_random_tie_breaking` that was called in the loop over training samples. But no need now everything is broadcasted !